tests/util/bft.py

```diff
@@ -393,6 +393,46 @@ def force_quorum_including_replica(self, replica_id, primary=0):
                     nursery.start_soon(self._assert_state_transfer_not_started,
                                        r)
 
+    async def assert_fast_path_prevalent(self, as_of_seq_num):
+        metric_key = ['replica', 'Counters', 'slowPathCount']
+        nb_slow_paths = await self.metrics.get(0, *metric_key)
+
+        assert nb_slow_paths - as_of_seq_num <= 1, \
```

Comment:
I don't follow what you are trying to show here. Is it that there are more fast paths messages than slow path messages at a given seq number? A doc comment would probably help.

Reply:
Agreed - both the implementation and the doc need improvement.